<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleCalibration</class>
 <widget class="QWidget" name="ScaleCalibration">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>570</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>660</y>
     <width>121</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxDatas">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>191</width>
     <height>221</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEditPhyDist">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPhyDistTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>物理距离</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPixDist">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPixDistTitle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>像素距离</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPhyDistUnit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPixDistUnit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>40</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>pixel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRatio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelRatioTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>像素比</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxCoordinate">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>530</y>
     <width>190</width>
     <height>221</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btnMoveRight">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>右</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveDown">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>下</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>左</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMoveUp">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>上</string>
    </property>
    <property name="autoRepeat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelP1Coordinate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>(100,100)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonP1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>点1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonP2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>点2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelP2Coordinate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>(100,100)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>刻度校准</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
